src/NServiceBus.Core/Transports/Msmq/MsmqTransportInfrastructure.cs

```diff
@@ -81,20 +81,18 @@ public override TransportReceiveInfrastructure ConfigureReceiveInfrastructure()
         {
             new CheckMachineNameForComplianceWithDtcLimitation().Check();
 
-            var builder = connectionString != null
-                ? new MsmqConnectionStringBuilder(connectionString).RetrieveSettings()
```

Comment:
`ConfigureSendInfrastructure` later in this class makes this call again. Should it be updated to retrieve the one from the settings collection?